charts/zot/values.yaml

```diff
@@ -18,6 +18,13 @@ serviceAccount:
   # If not set and create is true, a name is generated using the fullname template
   name: ""
 service:
+  headless:
+    # Enable headless service, only for statefulset
+    enabled: true
```

Comment:
I see the upgrade test is failing. Shouldn't this be disabled by default? Maybe it is related?